tests/e2e/utils/DevWorkspaceConfigurationHelper.ts

```diff
@@ -4,6 +4,8 @@ import { V1alpha2DevWorkspaceTemplate } from '@devfile/api';
 import YAML from 'yaml';
 import * as axios from 'axios';
 import { Logger } from './Logger';
+import { TestConstants } from '../constants/TestConstants';
+import { ShellExecutor } from './ShellExecutor';
 
 interface IContextParams {
```

Comment:
It's not clear what `IContextParams` is used for in the tests.
E.g., I can't find the usage of injectDefaultComponent and defaultComponentImage in API tests.

What do you think about adding a comment to the code with more details?

Reply:
It used to see which exactly fields you can pass into `new DevWorkspaceConfigurationHelper(params);` as params:
![image](https://github.com/eclipse/che/assets/72726250/e5af3333-30e8-458a-8f8c-919955fdc7b2)

Reply:
Here is used `injectDefaultComponent` and `defaultComponentImage`, it described in DevWorkspace genetatortool read me
![image](https://github.com/eclipse/che/assets/72726250/268157a3-7729-44ae-bf13-5e3472369b61)

Reply:
I added link to readme as comment